cli/src/main.rs

```diff
@@ -987,6 +989,41 @@ struct ExpandCluster {
     disk_type: Option<String>,
 }
 
+#[derive(Debug, StructOpt)]
+#[structopt(about = "Resize a cluster")]
+struct ResizeCluster {
+    #[structopt(long, parse(try_from_str = parse_org_id), default_value = "", help = "The organization id the cluster relates to")]
+    org_id: OrgId,
+
+    #[structopt(long, parse(try_from_str = parse_project_id), default_value = "", help = "The project id the cluster relates to")]
+    project_id: esc_api::resources::ProjectId,
+
+    #[structopt(long, short, parse(try_from_str = parse_cluster_id), help = "Id of the cluster you want to resize")]
+    id: esc_api::ClusterId,
+
+    #[structopt(long, help = "The target instance size. (C4, M8, etc)")]
+    target_size: String,
+}
+
+#[derive(Debug, StructOpt)]
+#[structopt(about = "Upgrade a cluster")]
+struct UpgradeCluster {
+    #[structopt(long, parse(try_from_str = parse_org_id), default_value = "", help = "The organization id the cluster relates to")]
+    org_id: OrgId,
+
+    #[structopt(long, parse(try_from_str = parse_project_id), default_value = "", help = "The project id the cluster relates to")]
+    project_id: esc_api::resources::ProjectId,
+
+    #[structopt(long, short, parse(try_from_str = parse_cluster_id), help = "Id of the cluster you want to upgrade")]
+    id: esc_api::ClusterId,
+
+    #[structopt(
+        long,
+        help = "The target tag you want to upgrade to. This must include the full version (23.10.1)."
```

Comment:
This is a little unclear (but not a blocker):
```suggestion
        help = "The target tag you want to upgrade to. This must include the full version (example: \"23.10.1\")."
```